src/crags/loaders/route-comments.loader.ts

```diff
@@ -16,7 +16,7 @@ export class RouteCommentsLoader implements NestDataLoader<string, Comment[]> {
 
```

Comment:
where loader is getting comments, should pass in current user, to filter them by publish_status and is_hidden